plugins/inmemoryscanner/test/Scanner_unittest.cpp

```diff
@@ -232,8 +242,13 @@ namespace InMemoryScanner
                     memoryRegions->push_back(std::move(memoryRegionDescriptor));
                     return memoryRegions;
                 });
-        auto expectedFileNameRegEx = trimmedProcessName + "-" + std::to_string(pid) + "-" + protectionAsString + "-" +
-                                     intToHex(startAddress) + "-" + intToHex(startAddress + size) + "-" + uidRegEx;
+        auto expectedFileNameRegEx = fmt::format("{}-{}-{}-{:x}-{:x}-{}",
```

Comment:
nit: why not use the getMemFileName function?

Reply:
I do not want to use a function from production in order to set up the premise for test results.

Reply:
this is a unittest file :-D

Reply:
nevermind
uuid is a regex here